go/types.go

```diff
@@ -229,13 +229,12 @@ type SignedTreeHead struct {
 }
 
 // SignedCertificateTimestamp represents the structure returned by the
-// add-chain and add-pre-chain methods after base64 decoding. (see RFC sections
-// 3.2 ,4.1 and 4.2)
+// add-chain and add-pre-chain methods after base64 decoding; see sections
+// 3.2, 4.1 and 4.2.
 type SignedCertificateTimestamp struct {
-	SCTVersion Version    // The version of the protocol to which the SCT conforms
-	LogID      SHA256Hash // the SHA-256 hash of the log's public key, calculated over
-	// the DER encoding of the key represented as SubjectPublicKeyInfo.
-	Timestamp  uint64          // Timestamp (in ms since unix epoc) at which the SCT was issued
+	SCTVersion Version         // The version of the protocol to which the SCT conforms
+	LogID      SHA256Hash      // The SHA-256 hash of the (DER-encoded) public key for the Log.
```

Comment:
You've been meticulous about consistency of having full stops at the end of comments; should these field comments have full stops too, or not?  Either there is a stray full stop on this line, or the others also need them :)